src/plugins/unified_search/public/query_string_input/text_based_languages_editor/esql_documentation_sections.tsx

```diff
@@ -14,13 +14,23 @@ export const initialSection = (
     markdown={i18n.translate(
       'unifiedSearch.query.textBasedLanguagesEditor.documentationESQL.markdown',
       {
-        defaultMessage: `## How it works
-Elasticsearch ESQL is a piped language where you can combine aggregations, filters, transformations, and projections in a single expression.
-                    
-ESQL query example:
+        defaultMessage: `## ESQL
+The Elasticsearch Query Language (ESQL) is a query language that enables the iterative exploration of data.
+
+An ESQL query consists of a series of commands, separated by pipes. Each query starts with a source command. A source command produces a table, typically with data from Elasticsearch. 
+
+A source command can be followed by one or more processing commands. Processing commands change an input table by adding, removing, or changing rows and columns. 
+
+You can chain processing commands, separated by a pipe character: \`|\`. Each processing command works on the output table of the previous command. 
```

Comment:
MInor suggestion.

```suggestion
To chain processing commands, use the pipe character: \`|\`. Each processing command works on the output table of the previous command. 
```

Reply:
@abdonpijpelink Can you please include a screenshot of the help?

Reply:
Is there a specific screenshot you're looking for? This is what it looks like when you open the help: 

<img width="1194" alt="image" src="https://user-images.githubusercontent.com/9715543/233014817-5894cae6-5372-4a72-9a7c-5c6d9bf66951.png">